web/src/AdapterEditPage.js

```diff
@@ -131,7 +131,7 @@ class AdapterEditPage extends React.Component {
         </Row>
         <Row style={{marginTop: "20px"}} >
           <Col style={{marginTop: "5px"}} span={(Setting.isMobile()) ? 22 : 2}>
-            {Setting.getLabel(i18next.t("provider:Type"), i18next.t("provider:Type - Tooltip"))} :
+            {Setting.getLabel(i18next.t("adapter:Type"), i18next.t("adapter:Type - Tooltip"))} :
```

Comment:
This is not correct fix and it's also not what we want either. This issue: https://github.com/casdoor/casdoor/issues/1566 asked to fix all Tooltip texts in all pages, not only adapter page. And we should fix English and Chinese first